### ISSUE TITLE
Report templates made using earlier versions of squid3 cannot be imported by current version

### ISSUE THREAD

**Issue by NicoleRayner** [CIRDLES/Squid#602]
I made the attached report template using v1.6.7 (or thereabouts) and I am unable to import them in version 1.7 or more recent.  It imports just fine using v1.6.7.  At some point between 1.6.7 and 1.7 a change was made so that CUSTOM expressions would also output their errors in the report so I suspect that might be part of the reason why the current version will not support the old templates.

Ideally we would have backwards capability in report templates, but if that is not possible, we at least need some kind of error message to indicate the problem (currently after import, it just does nothing - it doesn't hang give any indication of a problem, it just pretends the import never happened)

[SQUID2.5_GSC_StandardData.zip](https://github.com/CIRDLES/Squid/files/6159791/SQUID2.5_GSC_StandardData.zip)

I tested this by making a new model in v1.74, and I was able to import that one without any problems, while the one attached still would not import.

**Comment by bowring** [comment]
The next version of Squid will feature xml schema for reports and tasks, thanks to @drakene , and then this report will be readable.    A workaround in the short term is to open your squid project in 1.6.7, import the report, save the project, and then open it in 1.7n.

**Comment by NicoleRayner** [comment]
This is not yet fixed in v1.7.6.  It is not recognizing my custom tasks built in previous versions.  I don't ever recall this being a problem before (like it was for report templates when we went from v1.6 to 1.7).  I'm certain I have loaded custom tasks successfully regardless of version.   I can use the same workaround as suggested above (use task in project 1.7.4, save it, open it in 1.7.6 then save that same task as xml, then it is recognized moving forward) but that is not ideal.  
[Zircon_GSC_11pk_(Fixed_slope)_with_MF_corr.zip](https://github.com/CIRDLES/Squid/files/6360221/Zircon_GSC_11pk_.Fixed_slope._with_MF_corr.zip)

I'm not sure if it is working properly for templates, as I am not certain of the vintage of my custom templates at the moment.  I think it is the exact same as the one linked to above - and those I can import no problem into the latest version.

**Comment by bowring** [comment]
@NicoleRayner -  you are conflating reports and tasks a little, but
1) the legacy task XML files can be repaired in the short term by removing the line "" at the top of the file.  The next release will remove this line if it exists.
2) the legacy report file you have has to be repaired by hand:
the end of the file must be either
```xml
  </reportCategories>
  <reportSpotTarget>REFERENCE_MATERIAL</reportSpotTarget>
  <isDefault>false</isDefault>
  <version>0</version>
</SquidReportTable>
```
OR
```xml
  </reportCategories>
  <reportSpotTarget>UNKNOWN</reportSpotTarget>
  <isDefault>false</isDefault>
  <version>0</version>
</SquidReportTable>```


Thanks for testing!
